contracts/DLCManager.sol

```diff
@@ -258,6 +266,41 @@ contract DLCManager is
         return false;
     }
 
+    /**
+     * @notice  Checks mint eligibility.
+     * @dev     Checks if the amount is non-zero.
+     * @dev     If PoR is disabled, returns true.
+     * @dev     If PoR is enabled, checks if the new total value minted is within bounds.
+     * @dev     If the PoR check fails, reverts with an error.
+     * @param   amount  dlcBTC to mint.
+     * @param   currentTotalMinted  total minted value in all vaults.
```

Comment:
might want to just mention here 'in all vaults on this chain' or maybe that's weird because ofc this contract is on this chain?